docs/guides/02-using-style-helpers.md

```diff
@@ -0,0 +1,231 @@
+        id="using-style-helpers-step-3"
+        src="/developers/web-sdk/examples/maps/guides/using-style-helpers/step-3.html"
+        width="100%"
+        height="500"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+</div>
+> View this step [here](/developers/web-sdk/examples/maps/guides/using-style-helpers/step-3.html)
+
+Now we are going to use the `carto.viz.style.colorCategories` helper to assign a different color to each country depending on the `continent` feature property. We can leave empty the `options` object or we can specify one or more parameters. Please go to the [reference](ToDo Add link) to view the full list of options and default values for this helper. In this example, we are only specifying the color palette. 
+
+```js
+const countriesColorStyle = new carto.viz.style.colorCategories('continent', { palette: 'teal' });
+const countriesLayer = new carto.viz.Layer('ne_50m_admin_0_countries', countriesColorStyle);
+```
+
+<div class="example-map">
+    <iframe
+        id="using-style-helpers-step-4"
+        src="/developers/web-sdk/examples/maps/guides/using-style-helpers/step-4.html"
+        width="100%"
+        height="500"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+</div>
+> View this step [here](/developers/web-sdk/examples/maps/guides/using-style-helpers/step-4.html)
+
+Finally we are going to use the `carto.viz.style.colorContinuous` helper to create a continuous color ramp depending on the country GDP. This time we are not sending any parameter so we will use the default values. Please go to the [reference](ToDo Add link) to view the full list of options and default values for this helper.  
+
+```js
+const countriesColorStyle = new carto.viz.style.colorContinuous('gdp_md_est');
+const countriesLayer = new carto.viz.Layer('ne_50m_admin_0_countries', countriesColorStyle);
+```
+
+<div class="example-map">
+    <iframe
+        id="using-style-helpers-step-5"
+        src="/developers/web-sdk/examples/maps/guides/using-style-helpers/step-5.html"
+        width="100%"
+        height="500"
+        style="margin: 20px auto !important"
+        frameBorder="0">
+    </iframe>
+</div>
+> View this step [here](/developers/web-sdk/examples/maps/guides/using-style-helpers/step-5.html)
+
+### Size style helpers
+
+Similar to the color style helpers, you can use the size style helpers to easily change the size ot the symbology representing the features depending on the values of a given feature property. If the property data type is numeric, you can group the features in bins using the `carto.viz.style.sizeBins` helper. If the property data type is a string representing values from a data domain, you can group the features in categories using the `carto.viz.style.sizeCategories` helper. There is an additional helper `carto.viz.style.sizeContinuous` that you can use for creating a continuous color ramp for numeric feature properties.
```

Comment:
typo in "continuous color ramp" --> "continuous size" + "interpolation" ?

Reply:
Fixed